abl/util/lockfile.py

```diff
@@ -60,7 +60,8 @@ def __enter__(self):
             else:
                 fcntl.flock(self.file, lock_flags)
         except IOError, e:
-            if e[0] == errno.EAGAIN:
+            error_code = errno.EACCES if is_windows else errno.EAGAIN
```

Comment:
@uucidl good catch. Do you see any fault in just not predicating, and instead checking for "code in (EACCESS, EAGAIN)" - that should cover all options? I checked that the respective other is not used by windows/bsd to signal *other* conditions.